src/tests/integration/test_nycdb.py

```diff
@@ -14,10 +14,10 @@
 data_dir = os.path.join(os.path.dirname(__file__), 'data')
 
 ARGS = SimpleNamespace(
-    user=os.environ.get('NYCDB_TEST_POSTGRES_USER', 'postgres'),
-    password=os.environ.get('NYCDB_TEST_POSTGRES_PASSWORD', 'password'),
+    user=os.environ.get('NYCDB_TEST_POSTGRES_USER', 'nycdb'),
```

Comment:
i changed this fallback values to match the environment variables set in the `Dockerfile` and now the tests work